From 0c632eb5d6e19a8ebafb206946771eabbd25c3d7 Mon Sep 17 00:00:00 2001
From: Brandon Maier <brandon.maier@gmail.com>
Date: Sun, 17 Nov 2024 15:07:17 -0600
Subject: [PATCH] pylibfdt: remote DESTDIR from setup.py install

Signed-off-by: Brandon Maier <brandon.maier@gmail.com>
---
 pylibfdt/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pylibfdt/meson.build b/pylibfdt/meson.build
index 6157f8d..5a288c9 100644
--- a/pylibfdt/meson.build
+++ b/pylibfdt/meson.build
@@ -10,4 +10,4 @@ pylibfdt = custom_target(
   build_by_default: true,
 )
 
-meson.add_install_script(setup_py, 'install', '--prefix=' + get_option('prefix'), '--root=$DESTDIR')
+meson.add_install_script(setup_py, 'install', '--prefix=' + get_option('prefix'))
-- 
2.47.0

